### PR TITLE
Export updated page config from production.

### PR DESCRIPTION
**Pull request by Brian-Clement**
Exports some configuration performed manually on the production site.

### CODE REVIEW

**Comment by froboy** [comment]
@Brian-Clement I just made a whole bunch more changes on prod... you'll probably want to update this. :) is there documentation of the process for getting changes from prod?

**Comment by Brian-Clement** [comment]
@froboy there's no documentation that I'm aware of.  I pretty much just sync the production db down locally and export.  I'll get this updated with the latest from prod and get it in before anything else changes!

File: conf/drupal/config/block.block.views_block__event_sponsors_block_8.yml [hunk 1]
```diff
@@ -26,6 +26,6 @@ settings:
 visibility:
   request_path:
     id: request_path
-    pages: '/*/sponsors'
```
Comment (Brian-Clement) [inline conf/drupal/config/block.block.views_block__event_sponsors_block_8.yml]: This has removed the Contribution Day sponsors block from the `/2018/sponsors` page.

File: conf/drupal/config/block.block.views_block__event_sponsors_block_5.yml [hunk 2]
```diff
@@ -26,6 +26,6 @@ settings:
 visibility:
   request_path:
     id: request_path
-    pages: '/*/sponsors'
+    pages: /2019/sponsors
```
Comment (Brian-Clement) [inline conf/drupal/config/block.block.views_block__event_sponsors_block_5.yml]: This has removed the Party sponsors block from the `/2018/sponsors` page.

File: conf/drupal/config/block.block.views_block__event_sponsors_block_2.yml [hunk 3]
```diff
@@ -26,6 +26,6 @@ settings:
 visibility:
   request_path:
     id: request_path
-    pages: '/*/sponsors'
+    pages: /2019/sponsors
```
Comment (Brian-Clement) [inline conf/drupal/config/block.block.views_block__event_sponsors_block_2.yml]: This has removed the Contrib sponsors block from the `/2018/sponsors` page.

File: conf/drupal/config/block.block.views_block__event_sponsors_block_5.yml (hunk 2 above)
Comment (froboy) [reply conf/drupal/config/block.block.views_block__event_sponsors_block_5.yml]: I've corrected this in production. My intent was just to remove it from 2020 forward, but I forgot that 2018 was there.

File: conf/drupal/config/block.block.views_block__event_sponsors_block_2.yml (hunk 3 above)
Comment (froboy) [reply conf/drupal/config/block.block.views_block__event_sponsors_block_2.yml]: I've corrected this in production. My intent was just to remove it from 2020 forward, but I forgot that 2018 was there.

File: conf/drupal/config/block.block.views_block__event_sponsors_block_8.yml (hunk 1 above)
Comment (froboy) [reply conf/drupal/config/block.block.views_block__event_sponsors_block_8.yml]: I've corrected this in production. My intent was just to remove it from 2020 forward, but I forgot that 2018 was there.

**Comment by Brian-Clement** [comment]
That sounds totally fine @froboy, I just wanted to make sure nothing was unintentional there 👍

**Pull request merged**